library/ZendSearch/Lucene/Index/Term.php

```diff
@@ -87,7 +87,7 @@ public static function getPrefix($str, $length)
                 }
             }
 
-            if ($prefixBytes + $charBytes > strlen($str)) {
+            if(!isset($str[$prefixBytes+$charBytes-1])) {
```

Comment:
Please fix CS here (PSR-2 should be preferred).

`if (! isset($str[$prefixBytes + $charBytes - 1])) {`